### PR TITLE
OCPBUGS#11815: Required Azure parameters

### PR DESCRIPTION
**Pull request by xenolinux**
Version(s): 4.11+

Issue: https://issues.redhat.com/browse/OCPBUGS-11815

Link to docs preview:  https://58693--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_azure/installing-azure-customizations.html#installation-configuration-parameters-additional-azure_installing-azure-customizations

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/installation-configuration-parameters.adoc [hunk 1]
```diff
@@ -1070,7 +1070,7 @@ Additional Azure configuration parameters are described in the following table:
 |String, for example `production_disk_encryption_set`.
 
 |`compute.platform.azure.osDisk.diskEncryptionSet.subscriptionId`
-|Optional. The ID of a disk encryption set in another Azure subscription. This secondary disk encryption set will be used to encrypt compute machines. By default, the installation program will use the disk encryption set from the Azure subscription ID that you provided to the installation program prompts.
+|The ID of a disk encryption set in another Azure subscription. This secondary disk encryption set will be used to encrypt compute machines. By default, the installation program will use the disk encryption set from the Azure subscription ID that you provided to the installation program prompts.
```
Comment (jinyunma) [inline modules/installation-configuration-parameters.adoc:1073]: Installer will not set default subscriptionId any more if use doesn't provide it. This field is required one and need user provide. So below statements should be removed.
`By default, the installation program will use the disk encryption set from the Azure subscription ID that you provided to the installation program prompts.`

File: modules/installation-configuration-parameters.adoc [hunk 2]
```diff
@@ -1086,7 +1086,7 @@ Additional Azure configuration parameters are described in the following table:
 |String, for example `production_disk_encryption_set`.
 
 |`controlPlane.platform.azure.osDisk.diskEncryptionSet.subscriptionId`
-|Optional. The ID of a disk encryption set in another Azure subscription. This secondary disk encryption set will be used to encrypt control plane machines. By default, the installation program will use the disk encryption set from the Azure subscription ID that you provided to the installation program prompts.
+|The ID of a disk encryption set in another Azure subscription. This secondary disk encryption set will be used to encrypt control plane machines. By default, the installation program will use the disk encryption set from the Azure subscription ID that you provided to the installation program prompts.
```
Comment (jinyunma) [inline modules/installation-configuration-parameters.adoc:1089]: Same as compute, here also needs to remove the statement of "`By default, the installation program will use the disk encryption set from the Azure subscription ID that you provided to the installation program prompts."`

Comment (jinyunma) [reply modules/installation-configuration-parameters.adoc:1089]: It sounds not accurate for `The ID of a disk encryption set in another Azure subscription`.
I checked more from explain of installer.
```
$ ./openshift-install explain installconfig.platform.azure.defaultMachinePlatform.osDisk.diskEncryptionSet.subscriptionId
KIND:     InstallConfig
VERSION:  v1

RESOURCE: <string>
  SubscriptionID defines the Azure subscription the disk encryption set is in.
```
Suggest to reword `The ID of a disk encryption set in another Azure subscription`, how about change to `The ID of the Azure subscription the disk encryption set is in`?

same update also need to be applied on `compute.platform.azure.osDisk.diskEncryptionSet.subscriptionId`

Comment (xenolinux) [reply modules/installation-configuration-parameters.adoc]: @jinyunma I updated the PR and reworded the `  SubscriptionID defines the Azure subscription the disk encryption set is in.` sentence. can you PTAL?

Comment (jinyunma) [reply modules/installation-configuration-parameters.adoc]: thanks for addressing this.

Add @jhixson74  to review
The doc changes the description of platform.azure.osDisk.diskEncryptionSet.subscriptionId
`The ID of a disk encryption set in another Azure subscription.`
=> 
`Defines the Azure subscription of the disk encryption set, where the disk encryption set resides`

@jhixson74 Do you think is it reasonable? thanks.

File: modules/installation-configuration-parameters.adoc [hunk 3]
```diff
@@ -1070,7 +1070,7 @@ Additional Azure configuration parameters are described in the following table:
 |String, for example `production_disk_encryption_set`.
 
 |`compute.platform.azure.osDisk.diskEncryptionSet.subscriptionId`
-|Optional. The ID of a disk encryption set in another Azure subscription. This secondary disk encryption set will be used to encrypt compute machines. By default, the installation program will use the disk encryption set from the Azure subscription ID that you provided to the installation program prompts.
+|Defines the Azure subscription of the disk encryption set, where the disk encryption set resides. This secondary disk encryption set is used to encrypt compute machines.
```
Comment (kcarmichael08) [inline modules/installation-configuration-parameters.adoc:1073]: I don't think a comma is needed after "set". Otherwise lgtm!

**Review by skrthomas — COMMENTED** [review]
lgtm

**Pull request merged**